test/test_images/probe_helper/probe_helper_test.go

```diff
@@ -348,31 +318,23 @@ func TestProbeHelper(t *testing.T) {
 			}
 
 			// Set up resources for testing the CloudStorageSource.
-			gotRequest := make(chan requestData, 1)
-			hc, close := newTestServer(func(w http.ResponseWriter, r *http.Request) {
-				// The dummy Cloud Storage server forwards the client's generated HTTP requests.
-				body, _ := ioutil.ReadAll(r.Body)
-				gotRequest <- requestData{
-					method: r.Method,
-					url:    r.URL.String(),
-					body:   string(body),
-				}
-				w.Write([]byte("{}"))
-			})
-			defer close()
-			storageClient, err := storage.NewClient(ctx, option.WithHTTPClient(hc))
+			storageClient, gotRequest, cancel := testStorageClient(ctx, t)
```

Comment:
I think we can run these setup steps prior to each test case.

Reply:
Done.